app/controllers/ips_controller.rb

```diff
@@ -24,6 +28,12 @@ def index
 
 private
 
+  def current_locations
```

Comment:
Not sure this should be in the controller, also the name suggests to me that it is as useful as current_user or current organisation

Reply:
I'm no longer sure what we put in use cases and what not looking at our current state of controllers.

Reply:
This is a very view specific data transformation.
Not suggesting we put it in a use case.

Isn't there a more elegant way to do this without this intermediate transformation?
https://apidock.com/rails/v4.2.1/ActionView/Helpers/FormOptionsHelper/select